python/grass/jupyter/map3d.py

```diff
@@ -233,3 +233,10 @@ def show(self):
         from IPython.display import Image, display  # pylint: disable=import-outside-toplevel
 
         display(Image(self._filename))
+
+    def save(self, filename):
+        """Saves a PNG image of map3d to the specified *filename*"""
```

Comment:
This works for other formats than just PNG, no?

```suggestion
        """Saves an image to the specified *filename*"""
```